x-pack/platform/plugins/shared/alerting/server/routes/lib/validate_internal_rule_types_by_query.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RulesClient } from '../../rules_client';
+import { rulesClientMock } from '../../mocks';
+import { validateInternalRuleTypesByQuery } from './validate_internal_rule_types_by_query';
+
+describe('validateInternalRuleTypesByQuery', () => {
```

Comment:
nit: missing happy path test.

Reply:
There is a test called `should not throw an error for valid rule types`. Do you have something else on your mind?